atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/MultiCassandraStartupOrderingEteTest.java

```diff
@@ -138,22 +128,22 @@ private static void assertNotSatisfiedWithin(long time, Callable<Boolean> condit
                 .isInstanceOf(ConditionTimeoutException.class);
     }
 
-    private static boolean canAddTodo() {
+    private static boolean serverRunning() {
         try {
-            addATodo();
+            canPerformTransaction();
             return true;
         } catch (AtlasDbRemoteException e) {
+            return true;
+        } catch (Exception e) {
             return false;
         }
```

Comment:
Since `canPerformTransaction()` catches `AtlasDbRemoteException` and just returns false if that's the case, we don't need the `catch AtlasDbRemoteException` block here.

(I realise this is an issue with the original impl though!)